recipes/MLWC/meta.yaml

```diff
@@ -0,0 +1,59 @@
+
+{% set name = "MLWC" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://github.com/ToAmano/{{ name }}/archive/refs/tags/v.{{ version }}.tar.gz
+    sha256: 24c8ca4282551b975339ffe7d8b4b4adf80e7ca07290b31ead7f422c3162b0a8
+    
+build:
+  number: 0
+  noarch: python
+  script:
+    - {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation --prefix=$PREFIX
+
+requirements:
+  host:
+    - cmake
```

Comment:
This and boost below are typically only required foe compiled packages. Can you simply remove them?